### PR TITLE
Document trigger condition

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Document trigger condition

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51710

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by emontnemery** [comment]
Should we include complete automation examples, in addition to just examples of the trigger condition?
Do we need to be more explicit about this only being supported in automations?

File: source/_docs/scripts/conditions.markdown [hunk 1]
```diff
@@ -497,6 +497,36 @@ date part is fully ignored.
 
 </div>
 
+## Trigger condition
+
+The trigger condition can test if an automation was triggered by a certain trigger, identified by the trigger's `id`.
+
+```yaml
+condition: trigger
+  id: event_trigger
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]:
```suggestion
condition:
  condition: trigger
  id: event_trigger
```

File: source/_docs/scripts/conditions.markdown [hunk 2]
```diff
@@ -497,6 +497,36 @@ date part is fully ignored.
...
+```
+
+For a trigger identified by its index, both a string and integer is allowed:
+```yaml
+condition: trigger
+  id: "0"
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]:
```suggestion
condition:
  condition: trigger
  id: "0"
```

File: source/_docs/scripts/conditions.markdown [hunk 3]
```diff
@@ -497,6 +497,36 @@ date part is fully ignored.
...
+```
+
+```yaml
+condition: trigger
+  id: 0
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]:
```suggestion
condition:
  condition: trigger
  id: 0
```

File: source/_docs/scripts/conditions.markdown [hunk 4]
```diff
@@ -497,6 +497,36 @@ date part is fully ignored.
...
+```
+
+
+
+It is possible to give a list of triggers:
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]:
```suggestion

It is possible to give a list of triggers:

```

File: source/_docs/scripts/conditions.markdown [hunk 5]
```diff
@@ -497,6 +497,36 @@ date part is fully ignored.
...
+```yaml
+condition: trigger
+  id:
+    - event_1_trigger
+    - event_2_trigger
```
Comment (frenck) [inline source/_docs/scripts/conditions.markdown]:
```suggestion
condition:
  condition: trigger
  id:
    - event_1_trigger
    - event_2_trigger
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**